fundingmanager.go

```diff
@@ -681,6 +705,10 @@ func (f *fundingManager) initFundingWorkflow(targetPeer *peer, req *openChanReq)
 // wallet, then sends a funding request to the remote peer kicking off the
 // funding workflow.
 func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
+	if !f.checkMaxPending(msg.peer) {
```

Comment:
In this case, since a local RPC client initiated the request, an error should be returned via the provided error channel within the message. 

So: 
```
msg.err <- fmt.Errorf("some error you think of")
return
```